yb-voyager/cmd/analyzeSchema.go

```diff
@@ -361,6 +359,47 @@ func checkGist(sqlInfoArr []sqlInfo, fpath string) {
 	}
 }
 
+func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
```

Comment:
You can further divide it into functions for each check - checkStmtForGeneratedColumnsUsingParser()

Reply:
Agreed, let's further divide it into functions. 

```
if objtype == "TABLE"{
    createTableNode, ok := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateStmt)
    if ok {
        reportGeneratedAlwaysStoredColumns(createTableNode). 
        ...
}
```